### DISCUSSION TITLE
Inconsistent script addresses built with cardano-cli vs. PyCardano

### DISCUSSION THREAD

**Discussion by ajuggler** [Python-Cardano/pycardano#240] [Q&A] [answered]
Given the CBOR representation of a plutus validator, its script address can be easily generated with a variety of tools, e.g. cardano-cli and PyCardano. However, both tools generate different script addresses for the same CBOR. How is this possible? Am I finding a bug in PyCardano?

To illustrate the point, I describe an example where `cardano-cli` and `PyCardano` give different script addresses for the same CBOR in this post:

https://cardano.stackexchange.com/questions/11101/inconsistent-script-addresses-built-with-cardano-cli-vs-pycardano

**Comment by nielstron** [comment, marked as answer]
@michele-nuzzi did a nice answer on your stackexchange post :) https://cardano.stackexchange.com/a/11102

I think this is a general usability problem of pycardano and will open an issue to resolve this akin to https://github.com/OpShin/opshin/issues/123
